docs/writing-rules/pattern-syntax.mdx

```diff
@@ -266,6 +269,31 @@ if can_make_request:
 Half or partial statements can't be matches; both of the examples above must specify the contents of the condition’s body (e.g., `$BODY` or `...`), otherwise they are not valid patterns.
 :::
 
+### Matching single items with an ellipsis
+
+While ellipses generally stand for sequences of like elements, they
+are also supported in other contexts where they would match
+any single item. The following pattern is valid in languages with a C-like
+syntax even though `...` matches a single boolean value rather than a
+sequence:
```

Comment:
Note that I don't like the use of the term "operator". I'm not sure why we use it in this document. It's confusing because it's not used in the sense of "special syntax for a function application" as in `2 + 2` or `!cond`.

Reply:
@mjambon what term would you propose instead of operator?